exporter/jaeger/jaegerthrifthttpexporter/exporter_test.go

```diff
@@ -57,21 +50,27 @@ func TestNew(t *testing.T) {
 			},
 		},
 	}
+
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got, err := New(tt.args.config, tt.args.httpAddress, tt.args.headers, tt.args.timeout)
-			if (err != nil) != tt.wantErr {
-				t.Errorf("New() error = %v, wantErr %v", err, tt.wantErr)
-				return
-			}
-
-			if got == nil {
-				return
-			}
+			assert.NoError(t, err, "nil config")
```

Comment:
The third parameter of NoError looks incorrect to me. If I remember correctly it is the message to print when assertion fails. This assertion is not about config, why does it print "nil config"?

Reply:
Indeed, it is a wrong usage. Fixed